### PR TITLE
Individual area

### PR DESCRIPTION
**Pull request by ekr**
Here is an alternate proposal that *just* creates an individual space, with both addition and removal at the discretion of the chairs. I changed this from consensus to remove because it seems weird to have the chairs have discretion to adopt but not to remove and the primary reason for removal will be if it's just clearly out of scope.

I see that I've also removed the proposed principles for adoption. Those seem potentially reasonable but belong in another PR.

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -251,6 +250,35 @@ <h3 id=adopt>Document Adoption</h3>
...
+PATCG will set up and maintain <a
+href="https://github.com/patcg-individual-drafts">a separate space</a> with <a
+href="https://github.com/patcg/proposals" title="PATCG Proposals
+Repository">name and instructions defined in the proposals repository</a>. Any
+individual W3C or PATCG member may bring a proposal to that space with the
+understanding it will be governed by W3C rules and the <a
+href=https://www.w3.org/community/about/agreements/cla/>W3C Community
+Contributor License Agreement (CLA)</a>. The proposal should also have a notice
+on the `README.md` file at its base or on the document itself which clearly
+states "This document is an individual draft proposal. It and does not represent a work item of the Private Advertising Technology
```
Comment (cwilso) [inline charter.html:265]: 1) I'm not sure how the second sentence is supposed to be structured ("It and does not"). 2) Not sure why we're mentioning the (TBC) PATWG? 3) Is "work item" defined somewhere?  I'm not opposed to saying it's not one, just perhaps you really mean "this proposal has not been officially adopted by the PATCG" or something.

**Comment by npdoty** [comment]
Would we open issues for every individual draft in the same repo? Maybe just using a prefix in the issue title, or labels for an issue, so that we can filter issues to a particular individual draft.

File: charter.html [hunk 2]
```diff
@@ -251,6 +250,35 @@ <h3 id=adopt>Document Adoption</h3>
...
+states "This document is an individual draft proposal. It has not been
+adopted by the Private Advertising Technology Community Group."</p>
+
+<p>The document's author or <a href=#chairs>Chairs</a> may propose that the
+Community Group consider if a document in the space is potentially in scope. The
+<a href=#chairs>Chairs</a> then shall determine whether the document is <a
+href=#scope>in scope</a> for the Community Group and if so 
+it will be added to a list of relevant proposals for the community group's work
+<a href="https://github.com/patcg/proposals" title="PATCG Proposals repository">in the PATCG proposals repository</a>
+an create a repository under the individual contribution space.
```
Comment (jyasskin) [inline charter.html:274]:
```suggestion
and create a repository under the individual contribution space.
```

**Comment by ekr** [comment]
> Would we open issues for every individual draft in the same repo? Maybe just using a prefix in the issue title, or labels for an issue, so that we can filter issues to a particular individual draft.

This will be one draft per repo.

**Review by cwilso — APPROVED** [review]
LGTM

**Pull request merged**